README.md

```diff
@@ -1,31 +1,27 @@
-<!--a href="https://microclimate-dev2ops.github.io/">
-    <img src="mc-banner.png" alt="Microclimate Banner" height="75px" />
-</a-->
-
-# Codewind for VS Code
-
 [![Marketplace](https://img.shields.io/vscode-marketplace/v/IBM.codewind.svg?label=marketplace&logo=visual-studio-code)](https://marketplace.visualstudio.com/items?itemName=IBM.codewind)
 [![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/)
 [![Slack](https://img.shields.io/badge/ibm--cloud--tech-blue.svg?logo=slack&label=slack)](https://slack-invite-ibm-cloud-tech.mybluemix.net/)
 
-- **[Documentation](https://microclimate.dev/codewindtechpreview)**
-- **[Changelog](https://github.com/microclimate-dev2ops/codewind-vscode/blob/master/CHANGELOG.md)**
-
+# Codewind for VS Code
 Create and develop cloud-native, containerized web applications from VS Code.
 
-## Getting started
+## Installing Codewind
+Prerequisites
+- Install [VS Code](https://code.visualstudio.com/download).
+- Install Docker.
+- If you use Windows, you also need to install Docker Compose.
 
-1. Install [VS Code version 1.27 or later](https://code.visualstudio.com/download).
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+Complete all parts of the Codewind installation:
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or by searching for `Codewind` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions). Then, click the **Install** button.
+2. Go to the **Explorer** view group and open the **Codewind** view.
+3. To complete the installation, open the Codewind view in the **Explorer** view group or type `Focus on Codewind` into the Command Palette. A notification appears and asks if you would like to finish installing Codewind. 
+4. Click **Install** to complete the Codewind installation. The file is approximately 1 GB. When the installation completes, the server automatically starts.
```

Comment:
> The file is approximately 1 GB
it's not a file persay. it's a set of docker images. perhaps "The download is approximately 1 GB"